tutorials/cp-apm-nodejs-create-service/cp-apm-nodejs-create-service.md

```diff
@@ -423,34 +423,60 @@ Instead of using in-memory, you can also use persistent databases.
     [cds] - [ terminate with ^C ]
     ```
 
-### Test generic handlers with Postman
+### Test generic handlers
 
 You can now see the generic handlers shipped with CAP in action.
 
-1. Open the Postman application.
+Create a file called `test.http` in the root of your project and copy the following OData queries in it. This file can be used with the [REST client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) to make requests against your service. The generic handlers CAP provides sent the responses to your requests.
```

Comment:
Please first name the location and then the action.
Better say something like this: "In your projects root folder, create a file...".